samples/net/ipv4_autoconf/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _ipv4-autoconf-sample:
+
+Sample IPv4 autoconf client application
+#######################################
+
+Overview
+********
+
+This application starts a IPv4 autoconf and self-assigns
+a random IPv4 address in the 169.254.0.0/16 range, it defends
+the IPv4 address and resolves IPv4 conflicts if multiple
+parties try to allocate an identical address
+
+Requirements
+************
+
+- :ref:`networking_with_qemu`
+
+Building and Running
+********************
+
+These are instructions for how to use this sample application running on :ref:frdm_k64f board to configure a link local IPv4 address and connect to a Linux host.
```

Comment:
This looks too long line, could you split it. For generated documentation this does not matter but if reading via command line, this matters. Check also rest of the file for too long (>80 chars) lines.